src/main/java/cz/startnet/utils/pgdiff/PgDiffFunctions.java

```diff
@@ -68,7 +68,7 @@ public static void dropFunctions(final PrintWriter writer,
             if (!newSchema.containsFunction(oldFunction.getSignature())) {
                 searchPathHelper.outputSearchPath(writer);
                 writer.println();
-                writer.println(oldFunction.getDropSQL());
+                writer.println(oldFunction.getDropSQL(arguments));
```

Comment:
the majority of your changes are simply passing arguments all around the codebase